Doc/library/sqlite3.rst

```diff
@@ -957,6 +968,20 @@ Connection Objects
 Cursor Objects
 --------------
 
+   A cursor object represents a database cursor,
```

Comment:
Consider linking this with a reference summarizing what a database cursor is for the uninitiated (like me), such as

```suggestion
   A ``Cursor`` object represents a
   `database cursor <https://en.wikipedia.org/wiki/Cursor_(databases)>`_,
```